scripts/nym-node-setup/auto-bond/auto_bond_all.py

```
@@ -0,0 +1,124 @@
+        "--amount",                   row["amount"],
+        "--mnemonic",                 row["mnemonic"],
+        "--interval-operating-cost",  row["operator_cost"],
+        "--nyxd-url",                 NYXD_URL,
+        "--nym-api-url",              NYM_API_URL,
+        "-o", "json",
+    ])
+    if DRY_RUN:
+        return "DRY_RUN_PAYLOAD"
+    data = json.loads(result.stdout)
+    # payload is typically under a key like "payload" or "sign_payload"
+    return data.get("payload") or data.get("sign_payload") or list(data.values())[0]
```

Ruff (0.15.15)

[warning] Prefer `next(iter(data.values()))` over single element slice

Replace with `next(iter(data.values()))`

(RUF015)